System.IO.Abstractions/DriveInfoBase.cs

```diff
@@ -142,5 +142,12 @@ protected DriveInfoBase(IFileSystem fileSystem)
 
             return new DriveInfoWrapper(new FileSystem(), driveInfo);
         }
+
+        /// <summary>Returns a drive name as a string.</summary>
+        /// <returns>The name of the drive.</returns>
+        public override string ToString()
```

Comment:
To reduce public API changes and therefore downstream churn please undo the changes in `DriveInfoBase` and override `ToString` in `DriveInfoWrapper`/`MockDriveInfo` instead.

Reply:
Removed real implementation in test

Reply:
ToString moved to MockDriveInfo class